checker/src/main/java/org/checkerframework/checker/signature/SignatureAnnotatedTypeFactory.java

```diff
@@ -29,22 +30,32 @@
 /** Accounts for the effects of certain calls to String.replace. */
 public class SignatureAnnotatedTypeFactory extends BaseAnnotatedTypeFactory {
 
+    /** Annotation mirror for {@literal @}SignatureUnknown annotation */
     protected final AnnotationMirror SIGNATURE_UNKNOWN;
+    /** Annotation mirror for {@literal @}BinaryName annotation */
     protected final AnnotationMirror BINARY_NAME;
+    /** Annotation mirror for {@literal @}InternalForm annotation */
     protected final AnnotationMirror INTERNAL_FORM;
+    /** Annotation mirror for {@literal @}SourceNameForNonArrayNonInner annotation */
+    protected final AnnotationMirror SOURCE_NAME_FOR_NON_ARRAY_NON_INNER;
 
     /** The {@link String#replace(char, char)} method. */
     private final ExecutableElement replaceCharChar;
 
     /** The {@link String#replace(CharSequence, CharSequence)} method. */
     private final ExecutableElement replaceCharSequenceCharSequence;
 
+    /** The {@link String#contains(CharSequence)} method. */
+    private final ExecutableElement containsCharSequence;
```

Comment:
Please name this `stringContains`, because the containing class is more important than the parameter type and the `contains` method is not overloaded.
`replaceCharChar` contains its parameter types because the `replace` method is overloaded.